src/lib/gravityErrorHandler.ts

```diff
@@ -19,20 +25,35 @@ export const GravityMutationErrorType = new GraphQLObjectType<
     error: {
       type: GraphQLString,
     },
+    fieldErrors: {
+      type: GraphQLList(FieldErrorResultsType),
+    },
   }),
 })
 
 export const formatGravityError = (error) => {
-  const errorSplit = error.message.split(" - ")
+  const errorSplit = error.message?.split(" - ")
 
   if (errorSplit && errorSplit.length > 1) {
     try {
       const parsedError = JSON.parse(errorSplit[1])
-      if (parsedError.error) {
+      const { error, detail, text } = parsedError
+      const fieldErrorResults =
+        detail && Object.keys(pickBy(detail, isArray))?.length
```

Comment:
The second part of this `&&` feels a little opaque. Is it trying to check if `detail` contains any values that are arrays?

Reply:
Yes exactly - we are looking to see if any keys in this object have values that are arrays -- will add a comment.